### PR TITLE
CircleCI: Add dockerhub build

### PR DESCRIPTION
**Pull request by evaldofelipe**
Signed-off-by: Evaldo Felipe <contato@evaldofelipe.com>

**Motivation**

This change on CI allow us to keep the docker image updated on docker hub

### CODE REVIEW

**Review by chrismaree — COMMENTED** [review]
so the point of this is to get builds pushing back here: https://hub.docker.com/r/umaprotocol/protocol/tags

right?

seeing docker hub no longer does this on the free accounts?

will this block CI? if the build takes longer than running the unit tests will this block?

this will only build on merge to master and not on push to the branch, right?

File: packages/common/types/node-metamask/index.d.ts [hunk 1]
```diff
@@ -33,3 +33,5 @@ declare module "node-metamask" {
     };
 
 }
+
+// test comment
```
Comment (chrismaree) [inline packages/common/types/node-metamask/index.d.ts:37]: 👎

Comment (evaldofelipe) [reply packages/common/types/node-metamask/index.d.ts]: That was used to test changes on packages and be able to build a new image. Removed now after tests.

**Comment by evaldofelipe** [comment]
> this will only build on merge to master and not on push to the branch, right?
yes!

**Review by chrismaree — APPROVED** [review]

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**